keyboards/ergodox_ez/util/keymap_beautifier.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+import argparse
+import pycparser
+
+def beautify_src_file(filename):
+    src = {
+        "before": [],
+        "keys": [],
+        "after": []
+    }
+
+    with open(filename) as f:
+        current_section = "before"
+        for line in f.read().split("\n"):
+            if current_section == "before":
+                if line == "const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {":
+                    current_section = "keys"
+                    src[current_section].append("const int keymaps[]={")
+                else:
+                    src[current_section].append(line)
+            elif current_section == "keys":
+                src[current_section].append(line)
+                if line == "};":
+                    current_section = "after"
+            else:
+                src[current_section].append(line)
+
+    output = "\n".join(src["before"]) + "\n" 
+    output += beautify_keys_section("\n".join(src["keys"])) + "\n"
+    output += "\n".join(src["after"]) + "\n"
+    return output
```

Comment:
With the above changes, there's no need for `output` or `join`:
```suggestion
    return src['before'] + beautify_keys_section(src['keys']) + src['after']
```